provisioner/components/network/test_network_config.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
It will be good to have all test files in common place ( Inside provisioner ). If possible move these files to component specific test folder.

Reply:
> It will be good to have all test files in common place ( Inside provisioner ). If possible move these files to component specific test folder.

Done added test/unit and test/integration folder structure at provisioner level and moved network component test files there.

Reply:
Ok, Thanks!